community/duo_unix/APKBUILD

```diff
@@ -1,39 +1,38 @@
-# Contributor: Alan Lacerda <alacerda@alpinelinux.org>
-# Maintainer: Paul Morgan <jumanjiman@gmail.com>
+# Contributor: Kris Reese <ktreese@gmail.com>
+# Maintainer: Kris Reese <ktreese@gmail.com>
 pkgname=duo_unix
-pkgver=1.11.0
-pkgrel=1
-pkgdesc="duosecurity.com two-factor authentication"
-url="https://duo.com/support/documentation/duounix"
+pkgver=1.11.1
+pkgrel=0
+pkgdesc="Duo two-factor authentication for Unix systems with Pluggable Authentication Modules (PAM)"
+url="https://duo.com/docs/duounix"
 arch="all"
-license="GPL-2.0-only"
-depends=""
-depends_dev="openssl-dev zlib-dev"
-makedepends="$depends_dev automake autoconf libtool"
+license="GPL-2.0"
```

Comment:
I see that is GPL-2.0 plus a custom clause, so I guess it's more fitting than "GPL-2.0-only".

Reply:
works for me